bin/kibana.bat

```diff
@@ -22,7 +22,7 @@ If Not Exist "%NODE%" (
 )
 
 TITLE Kibana Server
-"%NODE%" %NODE_OPTIONS% "%DIR%\src\cli" %*
+"%NODE%" --max-old-space-size=256 %NODE_OPTIONS% "%DIR%\src\cli" %*
```

Comment:
Here's two ways.  I think the first is the more common (but longer);

```
@REM Loop through any command line parameters and if we find "--dev" add "--max-old-space-size=256"
:parse
IF "%~1"=="" GOTO endparse
IF "%~1"=="--dev" set NODE_OPTIONS="--max-old-space-size=256 %NODE_OPTIONS%"
SHIFT
GOTO parse
:endparse
```

And a single line way;

```
@REM If the command line parameters do not contain "--dev" then add "--max-old-space-size=256"
echo "%*" | findstr /V /C:"--dev" && set NODE_OPTION="--max-old-space-size=256 %NODE_OPTION%"
```